app/src/services/unwrap_token.ts

```diff
@@ -1,7 +1,7 @@
 import { utils } from 'ethers'
 import { BigNumber } from 'ethers/utils'
 
-import { wETHabi, wsPOAabi, wxDaiabi } from '../abi/wrapped_asset'
+import wrappedAsset from '../abi/wrappedAsset.json'
```

Comment:
Are they all the same ABI?

Reply:
Yeah I asked hexyl the same thing and wen't through them they looked same to me...